README.md

```diff
@@ -83,17 +83,17 @@ This extension supports a number of commands for interacting with supported serv
       
       If "args.override.boolean" is set to false, the server will attempt to auto-generate the launch arguments as it normally does when launched.
       
-   2. Can I run my Wildfly Server on a different port than the default one?
-      - Yes. To run a Wildfly Server on a different port you first have to edit the port in the standalone.xml file. 
+   ### 2. Can I run my Wildfly Server on a different port than the default one?
+      Yes. To run a Wildfly Server on a different port you first have to edit the port in the standalone.xml file. 
 
       The next step is to add the following setting through the Server Editor in VScode.
 
       Right-click your server -> Edit Server -> add "jboss.server.port": "8888". Change 8888 with the port you choose.
 
       Now if you start the server it should run on the specified port.
 
-   3. Is there a video that explain how the VSCode Server Connector extension and the Runtime Server Protocol work?
-      - Yes. This is the video you can watch to learn more about this extension [https://www.youtube.com/watch?v=sP2Hlw-C_7I] (https://www.youtube.com/watch?v=sP2Hlw-C_7I)
+   ### 3. Is there a video that explain how the VSCode Server Connector extension and the Runtime Server Protocol work?
+      Yes. This is the video you can watch to learn more about this extension https://www.youtube.com/watch?v=sP2Hlw-C_7I
```

Comment:
The video is awesome but we need to make sure that we are good from legal POV (music). @robstryker can you confirm

Reply:
The licenses on the used music are either completely free or just means the musician gets the add revenue. The main song is crab rave. License / details is here:  https://www.monstercat.com/licensing/content-creators